gorush/Dockerfile

```diff
@@ -3,10 +3,10 @@
 # Stage1: build from source
 FROM quay.io/cybozu/golang:1.15-bionic AS build
 
-ARG GORUSH_VERSION="1.12.0"
+ARG GORUSH_VERSION="1.13.0.3"
```

Comment:
I guess this is your fork and is a tentative release.
Why don't you tag it as `1.13.0.cybozu.1` or likewise?